### PR TITLE
Clarifies exitAfterDefer warning verbaige

### PR DESCRIPTION
**Pull request by josephzidell**
Clarifies the verbiage that the `exitAfterDefer` rule uses to warn the user to improve clarification as to the issue and consequences

### CODE REVIEW

File: checkers/testdata/_integration/check_main_only/linttest.golden [hunk 1]
```diff
@@ -49,4 +49,4 @@ exit status 1
 ./main.go:208:1: unlabelStmt: label loop is redundant
 ./main.go:216:11: unlambda: replace `func(x int) int { return add1(x) }` with `add1`
 ./main.go:219:39: unslice: could simplify xs[:] to xs
-./main.go:250:6: weakCond: suspicious `xs == nil || xs[0] == 0`; nil check may not be enough, check for len
\ No newline at end of file
+./main.go:250:6: weakCond: suspicious `xs == nil || xs[0] == 0`; nil check may not be enough, check for len
```
Comment (quasilyte) [inline checkers/testdata/_integration/check_main_only/linttest.golden:52]: Hmm, it looks like a trailing newline breaks our golden tests.
I'll send a fix that solves that so your code can pass the tests.

**Review by cristaloleg — APPROVED** [review]
<3

**Comment by quasilyte** [comment]
We need to merge #979 before this one.

**Comment by quasilyte** [comment]
I changed the merged commit description if you don't mind. :)

**Comment by josephzidell** [comment]
Thank you @quasilyte ! The new line was entered automatically by Github's online editor, and I could not remove it 😆 

Thanks for allowing this through and for merging it! 🍻

**Comment by quasilyte** [comment]
You're welcome. :)

**Pull request merged**